haystack/document_store/elasticsearch.py

```diff
@@ -48,7 +48,9 @@ def __init__(
         timeout=30,
         return_embedding: bool = False,
         duplicate_documents: str = 'overwrite',
-        index_type: str = "flat"
+        index_type: str = "flat",
+        synonyms=None,
+        synonym_type: str = "synonym"
     ):
```

Comment:
Can you please add the docstrings for the two newly introduced args (incl. the options for `synonym_type`, an example value for `synonyms` and the link to the elastic docs)?